types/simple-peer/index.d.ts

```diff
@@ -1,40 +1,77 @@
-        objectMode?: boolean; // set to true to create the stream in Object Mode. In this mode, incoming string data is not automatically converted to Buffer objects.
+        /** set to `true` if this is the initiating peer */
+        initiator?: boolean;
+        /** custom webrtc data channel configuration (used by [`createDataChannel`](https://developer.mozilla.org/en-US/docs/Web/API/RTCPeerConnection/createDataChannel)) */
+        channelConfig?: RTCDataChannelInit;
+        /** custom webrtc data channel name */
+        channelName?: string;
+        /** custom webrtc configuration (used by [`RTCPeerConnection`](https://developer.mozilla.org/en-US/docs/Web/API/RTCPeerConnection) constructor) */
+        config?: RTCConfiguration;
+        /** custom offer options (used by [`createOffer`](https://developer.mozilla.org/en-US/docs/Web/API/RTCPeerConnection/createOffer) method) */
+        offerOptions?: RTCOfferOptions;
+        /** custom answer options (used by [`createAnswer`](https://developer.mozilla.org/en-US/docs/Web/API/RTCPeerConnection/createAnswer) method) */
+        answerOptions?: RTCAnswerOptions;
+        /** function to transform the generated SDP signaling data (for advanced users) */
+        sdpTransform?: (this: Instance, sdp: string) => string;
+        /** if video/voice is desired, pass stream returned from [`getUserMedia`](https://developer.mozilla.org/en-US/docs/Web/API/MediaDevices/getUserMedia) */
+        stream?: MediaStream;
+        /** an array of MediaStreams returned from [`getUserMedia`](https://developer.mozilla.org/en-US/docs/Web/API/MediaDevices/getUserMedia) */
+        streams?: MediaStream[];
+        /**  set to `false` to disable [trickle ICE](http://webrtchacks.com/trickle-ice/) and get a single 'signal' event (slower) */
+        trickle?: boolean;
+        /** custom webrtc implementation, mainly useful in node to specify in the [wrtc](https://npmjs.com/package/wrtc) package. */
+        wrtc?: {
+            RTCPeerConnection: typeof RTCPeerConnection;
+            RTCSessionDescription: typeof RTCSessionDescription;
+            RTCIceCandidate: typeof RTCIceCandidate;
+        };
+        /** set to true to create the stream in Object Mode. In this mode, incoming string data is not automatically converted to Buffer objects. */
+        objectMode?: boolean;
     }
-
-    // https://github.com/feross/simple-peer/tree/v6.1.5#peer--new-simplepeeropts
     interface SimplePeer {
+        prototype: Instance;
+        /**
+         * Create a new WebRTC peer connection.
+         *
+         * A "data channel" for text/binary communication is always established, because it's cheap and often useful. For video/voice communication, pass the stream option.
+         *
+         * If opts is specified, then the default options (see <https://github.com/feross/simple-peer#peer--new-peeropts>) will be overridden.
+         */
         new (opts?: Options): Instance;
-        (opts?: Options): Instance;
```

Comment:
This is removed since it is declared as class in v9.6.0
https://github.com/feross/simple-peer/blob/e656b5ccd5ceac1d0bb636307e0a9ef4ac5d8259/index.js#L31